Core/Inc/nero_state_machine.h

```diff
@@ -0,0 +1,42 @@
+#ifndef NERO_STATE_MACHINE_H
```

Comment:
Why can't this just be called `nero.h` and `nero.c`? Is there really nothing else you'd want in the interface with NERO and MPU?